clustering.Rmd

```diff
@@ -181,33 +232,38 @@ In K-means clustering, we measure the quality of a cluster by its
 
 In the first cluster from the example, there are `r nrow(clus1)` data points. These are shown with their cluster center 
-(``r paste("flipper_length_scaled =", round(mean(clus1$flipper_length_scaled),2))`` and ``r paste("bill_length_scaled =", round(mean(clus1$bill_length_scaled),2))``) highlighted 
+(``r paste("flipper_length_standardized =", round(mean(clus1$flipper_length_standardized),2))`` and ``r paste("bill_length_standardized =", round(mean(clus1$bill_length_standardized),2))``) highlighted 
 in Figure \@ref(fig:10-toy-example-clus1-center).
 
-```{r 10-toy-example-clus1-center, echo = FALSE, warning = FALSE, fig.height = 4, fig.width = 4.35, fig.cap = "Cluster 1 from the toy example, with center highlighted."}
+```{r 10-toy-example-clus1-center, echo = FALSE, warning = FALSE, fig.height = 4, fig.width = 4.35, fig.cap = "Cluster 1 from the `penguin_data` data set example. Observations are in blue, with the cluster center highlighted in red."}
+base <- ggplot(data, aes(x = flipper_length_standardized, y = bill_length_standardized)) +
+  geom_point() +
+  xlab("Flipper Length (standardized)") +
+  ylab("Bill Length (standardized)")
+
 base <- ggplot(clus1) +
-  geom_point(aes(y = bill_length_scaled, x = flipper_length_scaled),  col = "dodgerblue3") +
-  labs(x = "Flipper Length (scaled)", y = "Bill Length (scaled)") +
+  geom_point(aes(y = bill_length_standardized, x = flipper_length_standardized),  col = "dodgerblue3") +
+  labs(x = "Flipper Length (standardized)", y = "Bill Length (standardized)") +
   xlim(c(
-    min(clus1$flipper_length_scaled) - 0.25 * sd(clus1$flipper_length_scaled),
-    max(clus1$flipper_length_scaled) + 0.25 * sd(clus1$flipper_length_scaled)
+    min(clus1$flipper_length_standardized) - 0.25 * sd(clus1$flipper_length_standardized),
+    max(clus1$flipper_length_standardized) + 0.25 * sd(clus1$flipper_length_standardized)
   )) +
   ylim(c(
-    min(clus1$bill_length_scaled) - 0.25 * sd(clus1$bill_length_scaled),
-    max(clus1$bill_length_scaled) + 0.25 * sd(clus1$bill_length_scaled)
+    min(clus1$bill_length_standardized) - 0.25 * sd(clus1$bill_length_standardized),
+    max(clus1$bill_length_standardized) + 0.25 * sd(clus1$bill_length_standardized)
   )) +
-  geom_point(aes(y = mean(bill_length_scaled), x = mean(flipper_length_scaled), colour = "red", size = 2.5)) +
+  geom_point(aes(y = mean(bill_length_standardized), x = mean(flipper_length_standardized), colour = "red", size = 2.5)) +
   theme(legend.position = "none") 
 base
 ```
 
 The second step in computing the WSSD is to add up the squared distance \index{distance!K-means} between each point in the cluster and the cluster center.
-We use the straight-line / Euclidean distance formula that we learned about in the classification chapter.
+We use the straight-line / Euclidean distance formula that we learned about in the [classification chapter](\@ref(classification)).
```

Comment:
Instead of "classification chapter" keep consistent with the other refs we use elsewhere:

"that we learned about in Chapter \\@ref(classification)"

Reply:
done